bcdb_test/performance/chart.html

```diff
@@ -0,0 +1,55 @@
+<canvas id="myChart"></canvas>
```

Comment:
i think if u used `matplotlib.pyplot` u will convert all the chart work into one function of simple lines :D unless u have a point of using the chart in this way?